src/types/interfaces/bill.ts

```
@@ -0,0 +1,65 @@
+	CustomerRef?: ReferenceType;
+	BillableStatus?: 'Billable' | 'NotBillable';
+	TaxCodeRef?: ReferenceType;
+}
+
+interface Line {
+	Id?: string;
+	Description?: string;
+	Amount?: number;
+	DetailType?: 'AccountBasedExpenseLineDetail' | 'ItemBasedExpenseLineDetail';
+	AccountBasedExpenseLineDetail?: AccountBasedExpenseLineDetail;
+	ItemBasedExpenseLineDetail?: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)